src/encoded/schemas/mixins.json

```diff
@@ -199,6 +199,17 @@
             "uniqueKey": true
         }
     },
+    "comments": {
+        "comments": {
+            "title": "Comments",
+            "comment": "Intended for general comments concerning an item.",
```

Comment:
You might want to indicate this is for internal use only, if that is indeed the intended use given permission = restricted_fields is set

Reply:
Yes, it is intended for internal use only. Modified the comment accordingly.